src/nunit.analyzers.tests/nunit.analyzers.tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RootNamespace>NUnit.Analyzers.Tests</RootNamespace>
-    <TargetFrameworks>net6.0;net462</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
This needs to be done conditionally on OS. We still want to test .net framework on Windows

Reply:
I agree with your suggestion.
Therefore, I changed only ci.yml like this. 
[552b72c](https://github.com/nunit/nunit.analyzers/pull/513/commits/552b72c686da6de7b06c53efe9f757c615c0432a)